### PR TITLE
Fix numpy version tests to use Looseversion

### LINKED ISSUE #6489: Test errors with numpy 1.11.1rc1
On Windows, Python 3.5.1, 64-bit:
```
numpy 1.11.1rc1
matplotlib 1.5.2rc2
<snip>
======================================================================
ERROR: test suite for <class 'matplotlib.tests.test_axes.transplant_class.<locals>.C'>
----------------------------------------------------------------------
Traceback (most recent call last):
  File "X:\Python35\lib\site-packages\nose\suite.py", line 210, in run
    self.setUp()
  File "X:\Python35\lib\site-packages\nose\suite.py", line 293, in setUp
    self.setupContext(ancestor)
  File "X:\Python35\lib\site-packages\nose\suite.py", line 316, in setupContext
    try_run(context, names)
  File "X:\Python35\lib\site-packages\nose\util.py", line 471, in try_run
    return func()
  File "X:\Python35\lib\site-packages\matplotlib\testing\decorators.py", line 154, in setup_class
    cls._func()
  File "X:\Python35\lib\site-packages\matplotlib\tests\test_axes.py", line 87, in test_formatter_large_small
    if tuple(map(int, np.__version__.split('.'))) >= (1, 11, 0):
ValueError: invalid literal for int() with base 10: '1rc1'

======================================================================
ERROR: test suite for <class 'matplotlib.tests.test_bbox_tight.transplant_class.<locals>.C'>
----------------------------------------------------------------------
Traceback (most recent call last):
  File "X:\Python35\lib\site-packages\nose\suite.py", line 210, in run
    self.setUp()
  File "X:\Python35\lib\site-packages\nose\suite.py", line 293, in setUp
    self.setupContext(ancestor)
  File "X:\Python35\lib\site-packages\nose\suite.py", line 316, in setupContext
    try_run(context, names)
  File "X:\Python35\lib\site-packages\nose\util.py", line 471, in try_run
    return func()
  File "X:\Python35\lib\site-packages\matplotlib\testing\decorators.py", line 154, in setup_class
    cls._func()
  File "X:\Python35\lib\site-packages\matplotlib\tests\test_bbox_tight.py", line 94, in test_bbox_inches_tight_raster
    if tuple(map(int, np.__version__.split('.'))) >= (1, 11, 0):
ValueError: invalid literal for int() with base 10: '1rc1'

----------------------------------------------------------------------
```

### PR DESCRIPTION
**Pull request by jenshnielsen**
Fixed #6489

### CODE REVIEW

File: lib/matplotlib/tests/test_axes.py [hunk 1]
```diff
@@ -84,7 +85,7 @@ def test_formatter_ticker():
 
 @image_comparison(baseline_images=["formatter_large_small"])
 def test_formatter_large_small():
-    if tuple(map(int, np.__version__.split('.'))) >= (1, 11, 0):
+    if LooseVersion(np.__version__) >= LooseVersion(1, 11, 0):
```
Comment (tacaswell) [inline lib/matplotlib/tests/test_axes.py]: I think this needs to be `LooseVersion('1.11.0')`

Comment (jenshnielsen) [inline lib/matplotlib/tests/test_axes.py]: I was sure that was what I typed Fixed now

**Comment by tacaswell** [comment]
Failure is one of the mathtext transients.

**Comment by tacaswell** [comment]
merged to 2.x while dropping these changes in b9873ab515fce612db47948fcd003f851b19d486

**Pull request merged**